crates/extensions/c8y_mapper_ext/src/actor.rs

```diff
@@ -127,11 +128,17 @@ impl C8yMapperActor {
             FsWatchEvent::DirectoryCreated(path) => {
                 if let Some(directory_name) = path.file_name() {
                     let child_id = directory_name.to_string_lossy().to_string();
-                    let message = Message::new(
-                        &Topic::new_unchecked(SMARTREST_PUBLISH_TOPIC),
-                        format!("101,{child_id},{child_id},thin-edge.io-child"),
-                    );
-                    self.mqtt_publisher.send(message).await?;
+                    let child_topic_id = EntityTopicId::default_child_device(&child_id).unwrap();
```

Comment:
This is the drawback of using *short* child device names in path: one has to assume a default topic scheme.

Ideally, I would use the same hierarchy for paths as for MQTT. i.e. with a `device` directory and there a sub directory per child plus a `main` directory. Or an `a/b/c/d` tree if the user prefer its own schema.

Reply:
> we could do that in a follow-up PR

Sure.